idx/cassandra/cassandra.go

```diff
@@ -497,3 +497,15 @@ func (c *CasIdx) prune() {
 		}
 	}
 }
+
+func (c *CasIdx) TagList(orgId int, from uint32) []string {
+	return c.MemoryIdx.TagList(orgId, from)
+}
+
+func (c *CasIdx) Tag(orgId int, tag string) map[string]uint32 {
+	return c.MemoryIdx.Tag(orgId, tag)
+}
+
+func (c *CasIdx) IdsByTagExpressions(orgId int, expressions []string) ([]string, error) {
+	return c.MemoryIdx.IdsByTagExpressions(orgId, expressions)
+}
```

Comment:
these additions should not be necessary. go compiler should automatically wire these through since MemoryIdx is embedded into the CasIdx struct (that's why Find, List, etc work also on this index)